trainingstorefront/web/addonsrc/trainingaddon/de/hybris/training/controllers/cms/CategoryProductsCarouselComponentController.java

```diff
@@ -27,7 +27,11 @@ protected void fillModel(HttpServletRequest request, Model model, CategoryProduc
         String categoryCode = component.getCategoryCode();
 
         final List<ProductData> products = new ArrayList<>();
-        products.addAll(trainingProductFacade.getProductsForCategory(categoryCode));
+
```

Comment:
Remember that files are copied from addon to he storefront, the you need to modify only the addon class, no both